index.js

```diff
@@ -140,9 +141,9 @@ export default function entrypoint(options = {}) {
 
       // Load existing app.yaml if it exists
       let yaml = {};
-      if (existsSync("app.yaml")) {
-        builder.log.minor("Existing app.yaml found");
-        yaml = YAML.parse(readFileSync("app.yaml").toString());
+      if (existsSync(defaultYamlFile)) {
```

Comment:
If a yaml file is specified i think the code should throw an error if the file is not found, otherwise it can probably be a bit confusing